docs/assignment1.md

```diff
@@ -1,62 +1,75 @@
+   				   Figure 1: A use case diagram of VUORK
+
+![VuORK use case diagram](https://lh3.googleusercontent.com/ePg0EoRTAvMiKXEP_VNm-gGz7HN0pBql_kOfZ-RUjralgX5VEYfBDSGg7DdKa8HeYwpv4C79SSpxdnjOcfLk6GLn7E-IDENFk6257sC3hyDslfFupreykPBXSfHrvP1NY3T5XnxJ)
+
+The remainder of this paper will cover detailed features and quality requirements to help us structure this project (This should be self explanatory). 
+
+## Features
+Author(s): Marta Jansone, Theresa Schantz
 
 ### Functional features
 
 As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
 
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+| ID   | Short name |                         Description                          |
+| ---- | ---------- | :----------------------------------------------------------: |
+| F1   | Commands   | The player can move the character using the following commands: *"move, take, put, drop, look, attack"*. The structure of this command will be <action> <items or objects>. This will help easily parse the input and help keep the commands easily comprehensible for the user. The command may include such articles as “the”, “a” or “an” and such prepositions as, for example, “in“ or “at”. |
+|      |            |                                                              |
+| F2   | Movements  | The player can move around in the game environment and get from one room to another by using the following commands: *"go/move followed by north, south, west, east, down, up, left and right"* |
+|      |            |                                                              |
+| F3   | Narrator   | Computer responding to user; describing what is happening, what the user is able to see |
+|      |            |                                                              |
+| F4   | Game-info  | The player can get the statistics of the game by entering the following commands… info, quit, inventory, time, diagnose |
+| F5   | Map        |    A list that keeps track of the places already visited     |
+| F6   | Backpack   | A list that keeps track of the items that have been picked up already |
+| F7   | Score      | An integer that shows the number of already obtained points; It will be updated after every action |
+| F8   | Moves      | An integer that shows the number of moves. Always updated after an action |
+| F9   | Timer      | A clock that shows the time that has already been passed since the start of the game |
```

Comment:
How does the moves and timer features affect the overall gameplay? how and what is their purpose in the game?